.circleci/config.yml

```diff
@@ -106,13 +106,13 @@ jobs:
       - node/install-packages
       - run:
           name: Run unit tests
-          command: npm run unit-test
+          command: npm run jest-unit-test -- --ci --maxWorkers=2
       - run:
           name: Run infra tests
-          command: npm run infra-test
+          command: npm run jest-infra-test -- --ci --maxWorkers=2
       - run:
           name: Run integration tests
-          command: npm run integration-test
+          command: npm run jest-integration-test -- --ci --maxWorkers=2
```

Comment:
Changed it to call jest directly so we don't run `build` with each. Also, saw weird issues with jest workers being killed in CircleCI, so reduced number of maxWorkers.

Reply:
How many workers was it running by default?